test/scripts/addEquals.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs')
```

Comment:
What's this file used for?

Reply:
The bash script you sent me required a few installs that didnt work, so it a script that adds =s to all tests. I also have been using it for mass JSON changes as I add a few new features to the Testing (e.g. I had to add deep vs normal comparators)

Reply:
(I saved it in a different file incase the new JSON was incorrectly formatted

Reply:
Ahhh! Okay no need to check this in.